src/lib/context.js

```diff
@@ -53,7 +57,7 @@ export class Context {
           () => langs);
   }
 
-  resolve(langs, id, args) {
+  _resolve(langs, id, args, formatter) {
     const lang = langs[0];
 
     if (!lang) {
```

Comment:
I can't put the comment under the specific line, but I think we'll need to differentiate here between value and object when we return `return { value: id, attrs: null };`.